### PR TITLE
Provide list of relevant vulnerability IDs in advisory summaries.

### PR DESCRIPTION
**Pull request by bobmcwhirter**
Ensure CVSS3 scores actually surface to the details
- fix bug related to fetching CVSS3 scores.

### CODE REVIEW

File: modules/graph/src/endpoints/advisory.rs [hunk 1]
```diff
@@ -4,44 +4,69 @@ use crate::graph::Graph;
...
-    ),
+context_path = "/api/v1/advisory",
+tag = "advisory",
+params(
+SearchOptions,
+Paginated,
+),
+responses(
+(status = 200, description = "Matching vulnerabilities", body = PaginatedAdvisorySummary),
+),
```
[the remark this answers is not in the document]
Comment (bobmcwhirter) [reply modules/graph/src/endpoints/advisory.rs:19]: I blame `cargo fmt`

**Comment by bobmcwhirter** [comment]
@carlosthe19916 also the CVSS scores? And the other assertions? Or is identifier+severity sufficient?

**Comment by carlosthe19916** [comment]
> @carlosthe19916 also the CVSS scores? And the other assertions? Or is identifier+severity sufficient?
identifier+severity sufficient? should be sufficient

File: modules/graph/src/endpoints/advisory.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (jcrossley3) [reply modules/graph/src/endpoints/advisory.rs]: I run that all the time. Never does what yours does.

**Review by carlosthe19916 — APPROVED** [review]
The `severity ` is not populated yet. But I am assuming that will come in another PR. Thanks for the enhancement @bobmcwhirter !

**Comment by bobmcwhirter** [comment]
score and severity will be coming shortly, lemme add that to this PR.

**Comment by bobmcwhirter** [comment]
Both `severity` and `score` should be populated now.

**Comment by carlosthe19916** [comment]
Thanks @bobmcwhirter . I confirm I can see the severity and score populated. I already approved this PR.

**Pull request merged**